### PR TITLE
[FIX] account: epd discount with foreign payment currency

### PR DESCRIPTION
**Pull request by agr-odoo**
Configure the 'Cash Discount Tax Reduction' to 'On Early Payment' Have a invoice in company curr
Add a payment term with early payment discount
Click 'Register Payment'
In the wizard select a foreign currency

Issue: No epd will be applied

opw-4111802

### CODE REVIEW

**Comment by agr-odoo** [comment]
@smetl 
Hello,
this is still a bit drafty, do we want/need this?

File: addons/account/tests/test_early_payment_discount.py [hunk 1]
```diff
@@ -226,6 +226,34 @@ def test_register_discounted_payment_on_single_out_invoice_with_tax(self):
...
+            'invoice_payment_term_id': self.early_pay_10_percents_10_days.id,
+        })
+        inv_1500_10_percents_discount_tax_incl_15_percents_tax.action_post()
+        active_ids = inv_1500_10_percents_discount_tax_incl_15_percents_tax.ids
+        wizard = self.env['account.payment.register'].with_context(active_model='account.move', active_ids=active_ids).create({
+            'payment_date': '2017-01-01',
+            'currency_id': self.currency_data['currency'].id,
+        })
+        payments = wizard._create_payments()
+        self.assertEqual(wizard.early_payment_discount_mode, True, "Early Payment mode not applied")
```
Comment (smetl) [inline addons/account/tests/test_early_payment_discount.py:246]: use `assertRecordValues`

File: addons/account/tests/test_early_payment_discount.py [hunk 2]
```diff
@@ -226,6 +226,34 @@ def test_register_discounted_payment_on_single_out_invoice_with_tax(self):
             {'amount_currency': 1552.5, 'tax_tag_invert': False},
         ])
 
+    def test_register_discounted_foreign_currency_payment_on_invoice_with_tax(self):
+        self.env.company.early_pay_discount_computation = 'included'
+        inv_1500_10_percents_discount_tax_incl_15_percents_tax = self.env['account.move'].create({
```
Comment (smetl) [inline addons/account/tests/test_early_payment_discount.py:231]:
```suggestion
        invoice = self.env['account.move'].create({
```
It's quite explicit it's an invoice-with-1500-but-a-tax-and-also-an-early-payment-but-price-included-blablabla

**Review by smetl — COMMENTED** [review]
@agr-odoo This is completely normal. We don't manage the EPD with a foreign currency.

**Pull request closed without merging**